hardware-types/pcb/rfq.json

```
@@ -0,0 +1,328 @@
+                                        "FR-1",
+                                        "FR-2",
+                                        "FR-3",
+                                        "FR-4",
+                                        "FR-5",
+                                        "CEM",
+                                        "CEM-1",
+                                        "CEM-2",
+                                        "CEM-3",
+                                        "Teflon",
+                                        "Polyimide",
+                                        "Duroid",
+                                        "Kapton",
+                                        "Polyester",
+                                        "Polyethylene",
+                                        "Polyflon",
+                                        "PTFE",
+                                        "Pyralux",
+                                        "XPC",
+                                        "Nelco 4000-13EPSI",
+                                        "Rogers 4350B",
+                                        "Panasonic",
+                                        "Megtron-6",
+                                        "Megtron-7"
+                                    ],
+                                    "type": "string"
+                                },
+                                "SurfaceFinish": {
+                                    "title": "Surface Finish",
+                                    "default": "HASL",
+                                    "enum": [
+                                        "ENIG",
+                                        "ENIPG",
+                                        "goldBare",
+                                        "carbon",
+                                        "copper",
+                                        "nickel",
+                                        "silver",
+                                        "solder",
+                                        "tin",
+                                        "HASL"
+                                    ],
+                                    "type": "string"
+                                }
+                            },
+                            "additionalProperties": false
+                        },
+                        "PCBSize": {
+                            "type": "object",
+                            "title": "PCB Dimensions",
+                            "required": [
+                                "Length",
+                                "Width",
+                                "Layers",
+                                "Thickness"
+                            ],
+                            "properties": {
+                                "Length": {
+                                    "title": "Length",
+                                    "default": 0.0,
+                                    "type": "number"
```

```suggestion
"default": 0,
"type": "integer"
"minimum": 0
```

I think all values should be in [SI (International System of Units)](https://en.wikipedia.org/wiki/International_System_of_Units) with "mm" (millimeters) being the smallest value.